README.md

```diff
@@ -16,7 +16,8 @@ Agent is thus strongly encouraged.
 To run as a javaagent [download the jar](https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.2.0/jmx_prometheus_javaagent-0.2.0.jar) and run:
 
 ```
-java -javaagent:./jmx_prometheus_javaagent-0.2.0.jar=1234:config.yaml -jar yourJar.jar
+# Serve metrics on port 8080
```

Comment:
This line is a bit distracting, what we usually do is include a link to the /metrics in the prose.

Reply:
Sure, will update